src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -6,7 +6,6 @@ import { CalciteLayout } from "../interfaces";
 
 /**
  * @slot action-bar - A slot for adding a `calcite-action-bar` to the panel.
- * @slot action-pad - A slot for adding a `calcite-action-pad` to the panel. The action pad will be positioned relative to the shell panel when displayed.
```

Comment:
question: 😱 should we consider breaking change?

Reply:
The slot is absent. Seems this doc line never got cleaned up. So I don't think it's a breaking change. just a Doc Typo.